src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DisposeMethodsShouldCallBaseClassDisposeTests.cs

```diff
@@ -1083,41 +1083,5 @@ End Sub
             // Test0.vb(14,26): warning CA2215: Ensure that method 'Sub B.Dispose()' calls 'MyBase.Dispose()' in all possible control flow paths.
             GetBasicResultAt(14, 26, "Sub B.Dispose()", "MyBase.Dispose()"));
         }
-
-        [Fact, WorkItem(1671, "https://github.com/dotnet/roslyn-analyzers/issues/1671")]
-        public async Task ErrorCase_NoDiagnostic()
-        {
-            // Missing "using System;" causes "Equals" method be marked as IsOverride but with null OverriddenMethod.
```

Comment:
This comment makes me think that this case *used to* cause a crash.

Reply:
Most likely, this tests the following code path:

https://github.com/dotnet/roslyn-analyzers/blob/b148fa89c974a72c28184931767835709f34d1fa/src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/DisposeMethodsShouldCallBaseClassDispose.cs#L55-L56

Reply:
> This comment makes me think that this case used to cause a crash

It may have, but not in this analyzer.